col/src/main/java/vct/col/ast/expr/MethodInvokation.scala

```diff
@@ -17,7 +17,7 @@ import scala.collection.JavaConverters
   * @author sccblom
   *
   */
-case class MethodInvokation(`object`: ASTNode, dispatch: ClassType, method: String, args: Seq[ASTNode]) extends ExpressionNode with VisitorHelper {
+case class MethodInvokation(`object`: ASTNode, dispatch: ClassType, method: String, var args: Seq[ASTNode]) extends ExpressionNode with VisitorHelper {
```

Comment:
Sorry Bob, but is 'var' some kind of modifier or so from scala? sorry my ignorance XD